Octopus.Client/PowerShell/Environments/AddEnvironments.ps1

```diff
@@ -30,7 +30,7 @@ foreach ($environmentName in $environments) {
             Name = $environmentName
         }
         
-        $repositoryForSpace.Environments.Create($environment)
+        $response = $repositoryForSpace.Environments.Create($environment)
```

Comment:
```suggestion
        $repositoryForSpace.Environments.Create($environment) | Out-Null
```

This would probably be better since the `$response` isn't actually being used?

Reply:
It is used on the next line. :)